src/components/CouponDetails/index.jsx

```diff
@@ -90,9 +90,34 @@ class CouponDetails extends React.Component {
     }
   }
 
+  getTableFilterSelectOptions() {
+    const { couponData: { usage_limitation: usageLimitation } } = this.props;
+    const shouldHidePartialRedeem = ['Single use', 'Once per customer'].includes(usageLimitation);
```

Comment:
Does JS like global constants at all? I know in Python I'd probably make 'Single Use' a variable that could be imported/referenced in case we change 'Single use' to 'Single Use' or something like that. Is that a thing that 1) people do? 2) is worth doing? Just a thought anyway!